go/flags/endtoend/mysqlctld.txt

```diff
@@ -22,6 +22,7 @@ Flags:
       --alsologtostderr                                                  log to standard error as well as files
       --app_idle_timeout duration                                        Idle timeout for app connections (default 1m0s)
       --app_pool_size int                                                Size of the connection pool for app connections (default 40)
+      --bind_address string                                              Bind address for the server.
```

Comment:
you'll need to generate the website docs for all these (`make generate-docs` in the website dir, see also the envvars you'll need to set)

Reply:
Got it! Thank you for the pointer @ajm188

Reply:
https://github.com/vitessio/website/pull/1600